### PR TITLE
CA-157 - First commit.

### PR DESCRIPTION
**Pull request by michael-kitchin**

### CODE REVIEW

File: src/features/wordDisplay/buttonWord.tsx [hunk 1]
```diff
@@ -81,16 +84,21 @@ export const ButtonWord = ({
...
                             borderStyle: 'none',
+                            ...(fillWidth ? { width: '100%' } : {}),
                             '.MuiButtonGroup-grouped': {
                               padding: '0px !important',
                               minWidth: '12px !important',
                               height: enableGlossDisplay ? '82px !important' : '62px !important'
-                            },
-                          }} >
-      {tokens?.map((token) => <ButtonToken key={token.id} token={token} completeWord={tokens} enableGlossDisplay={enableGlossDisplay} links={links} languageInfo={languageInfo} suppressAfter={suppressAfter} disabled={disabled} hoverHighlightingDisabled={disableHighlighting} />)}
+                              // ...(fillWidth ? { width: '100%' } : {})
```
Comment (blentz100) [inline src/features/wordDisplay/buttonWord.tsx:92]: Dead code I believe we should remove.

File: src/features/editor/index.tsx [hunk 2]
```diff
@@ -2,29 +2,43 @@
...
-const Editor = ({containers, position, usePaddingForEditorContainer}: EditorProps): ReactElement => {
+const Editor = ({
+                  containers,
+                  position,
+                  usePaddingForEditorContainer
+                }: EditorProps): ReactElement => {
   useDebug('Editor');
 
+  const [visibleSourceVerses, setVisibleSourceVerses] = useState<Verse[]>([]);
+  const setNewVisibleVerses = useCallback((inputVerses: Verse[], corpus: CorpusContainer) => {
```
Comment (blentz100) [inline src/features/editor/index.tsx:35]: Could you add a brief comment explaining what is happening inside this useCallback?

Comment (michael-kitchin) [reply src/features/editor/index.tsx:35]: Done.

File: src/features/wordDisplay/buttonWord.tsx (hunk 1 above)
Comment (michael-kitchin) [reply src/features/wordDisplay/buttonWord.tsx]: Good catch. Done.

**Review by blentz100 — APPROVED** [review]

**Pull request merged**